### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kooya3**
### Improving Documentation

necessary update for agilitycms readme file to enhance clarity

### CODE REVIEW

File: examples/cms-agilitycms/README.md [hunk 1]
```diff
@@ -74,19 +74,19 @@ First, [create an account on Agility CMS](https://agilitycms.com).
...
+1. Navigate to **Settings** > **Content Definitions**.
+2. Click on the "New" button to create a new Content Definition.
+3. In the Title field, enter "Author". The Reference Name will be automatically populated for you.
+4. Use the Form Builder tab to add the following fields:
+   - `Name`: Set the Field Label to "Name" and the Field Type to "Text".
+   - `Picture`: Set the Field Label to "Picture" and the Field Type to "Image".
+5. Once you have added these fields, click on "Save & Close" to save your Author content definition.
 
+By following these steps, you will be able to define an Author Content Definition in the Agility CMS Content Manager.
 ### Step 3. Create a `List` based on your `Author` Content Definition
```
Comment (brandonnorsworthy) [inline examples/cms-agilitycms/README.md:90]: proper documentation rules that you leave a new line before the next header section. You accidentally skipped it
```suggestion
By following these steps, you will be able to define an Author Content Definition in the Agility CMS Content Manager.

### Step 3. Create a `List` based on your `Author` Content Definition
```

**Review by kooya3 — COMMENTED** [review]
Updated the step 2 section with proper documentation rules to leave a new line before a header section

**Review by samcx — APPROVED** [review]
<picture data-single-emoji=":ty:" title=":ty:">:ty:</picture>

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 1f70732930894c55047f3e473a1ed990996208d4

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Pull request merged**